docs/Projects/TAD/ADRs/0002-tad-reporting-standard.md

```diff
@@ -0,0 +1,51 @@
+# TAD-0002 TAD Reporting Standard
+
+## Context
+
+The TAD Reporting Standard proposes a standardised way of capturing information of ML-models and systems.
+
+## Assumptions
+
+There does not exists a standard of capturing all relevant information on ML-models that also includes
+fairness and bias tests and regulatory assessments.
+
+A widely used implementation for [Model Cards for Model Reporting](https://arxiv.org/abs/1810.03993) is given
+by the [Hugging Face Model Card metadata specification](https://github.com/huggingface/hub-docs/blob/main/modelcard.md?plain=1),
+which in turn is based on [Papers with Code Model Index](https://github.com/paperswithcode/model-index).
+This implementation does not include fields for bias and fairness tests or regulartory assessments.
```

Comment:
```suggestion
This implementation does not include fields for measurements and metrics from technical tests like bias and fairness tests or regulatory assessments.
```

Reply:
It does include fields for metrics, only less elaborate than our metric field. How about: 
```suggestion
This implementation does not capture sufficient details about metrics and does not include measurements from technical tests on bias en fairness or regulatory assessments.
```
?

Reply:
Yes, very good, I can not edit your suggestion but it should say: 'bias and fairness'.. going Duch there ;)